apps/crn-frontend/src/network/teams/TeamProfile.tsx

```diff
@@ -17,14 +22,21 @@ const loadWorkspace = () =>
 
 const loadTeamOutput = () =>
   import(/* webpackChunkName: "network-team-team-output" */ './TeamOutput');
+const loadEvents = () =>
+  import(/* webpackChunkName: "network-upcoming-events" */ '../EventsEmbed');
```

Comment:
Not sure about this bundle name as I expect it will also be for past events too. Maybe call et network-events-embed or just network-events?